brick:
    outlier:
        mean_k: 50
        stddev_multiplier_thresh: 1

    kalman:
        noise_mv: 300
        noise_pos: 1
        noise_vel: 1

centroid_x:
    kalman:
        noise_mv: 1
        noise_pos: 1
        noise_vel: 1

centroid_y:
    kalman:
        noise_mv: 1
        noise_pos: 1
        noise_vel: 1

centroid_z:
    kalman:
        noise_mv: 300
        noise_pos: 1
        noise_vel: 1